tmp/build/patch-grafana.sh

```diff
@@ -63,6 +68,7 @@ function grafana_patch_deployment() {
 \2- name: GF_USERS_AUTO_ASSIGN_ORG_ROLE\
 \2  value: Admin\
 \1/' \
+  -e 's+image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"+image: "{{ .Values.global.hub }}/'${image}':{{ .Values.global.tag }}"+' \
```

Comment:
This would overwrite grafana-specific repository and tag values with global ones, is that what we want? I *just* needed a container-specific override on prometheus, hence the question.

Reply:
Lets wait for @brian-avery to comment since he was looking to standardise this